dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,229 @@
+class SentryStackFrame {
+  static final SentryStackFrame asynchronousGapFrameJson =
+      SentryStackFrame(absPath: '<asynchronous suspension>');
+
+  SentryStackFrame({
+    this.absPath,
+    this.fileName,
+    this.function,
+    this.module,
+    this.lineNo,
+    this.colNo,
+    this.contextLine,
+    this.inApp,
+    this.package,
+    this.native,
+    this.platform,
+    this.imageAddr,
+    this.symbolAddr,
+    this.instructionAddr,
+    this.rawFunction,
+    List<int> framesOmitted,
+    List<String> preContext,
+    List<String> postContext,
+    Map<String, String> vars,
+  })  : _framesOmitted = framesOmitted,
+        _preContext = preContext,
+        _postContext = postContext,
+        _vars = vars;
+
+  /// The absolute path to filename.
+  final String absPath;
+
+  final List<String> _preContext;
+
+  /// An immutable list of source code lines before context_line (in order) – usually [lineno - 5:lineno].
+  List<String> get preContext => List.unmodifiable(_preContext);
+
+  final List<String> _postContext;
+
+  /// An immutable list of source code lines after context_line (in order) – usually [lineno + 1:lineno + 5].
+  List<String> get postContext => List.unmodifiable(_postContext);
+
+  final Map<String, String> _vars;
+
+  /// An immutable mapping of variables which were available within this frame (usually context-locals).
+  Map<String, String> get vars => Map.unmodifiable(_vars);
+
+  final List<int> _framesOmitted;
+
+  /// Which frames were omitted, if any.
+  ///
+  /// If the list of frames is large, you can explicitly tell the system
+  /// that you’ve omitted a range of frames.
+  /// The frames_omitted must be a single tuple two values: start and end.
+  //
+  /// Example : If you only removed the 8th frame, the value would be (8, 9),
+  /// meaning it started at the 8th frame, and went untilthe 9th (the number of frames omitted is end-start).
+  /// The values should be based on a one-index.
+  List<int> get framesOmitted => List.unmodifiable(_framesOmitted);
+
+  /// The relative file path to the call.
+  final String fileName;
+
+  /// The name of the function being called.
+  final String function;
+
+  /// Platform-specific module path.
+  final String module;
+
+  /// The column number of the call
+  final int lineNo;
+
+  /// The column number of the call
+  final int colNo;
+
+  /// Source code in filename at line number.
+  final String contextLine;
+
+  /// Signifies whether this frame is related to the execution of the relevant code in this stacktrace.
+  ///
+  /// For example, the frames that might power the framework’s web server of your app are probably not relevant, however calls to the framework’s library once you start handling code likely are.
+  final bool inApp;
+
+  /// The "package" the frame was contained in.
+  final String package;
+
+  final bool native;
```

Comment:
is `native` a reserved word in Dart? or just GH making it red?

Reply:
seems it's just GH